app/assets/javascripts/components/common/article_viewer.jsx

```diff
@@ -69,7 +70,7 @@ const ArticleViewer = React.createClass({
   },
 
   isEnWiki() {
-    return this.props.article.language === 'en' && this.props.article.project === 'wikipedia';
+    return SUPPORTED_LANG.includes(this.props.article.language) && this.props.article.project === 'wikipedia';
```

Comment:
The name of this method should be changed; it no longer answers the question of whether it's English Wikipedia, but rather whether it's a wiki with whocolor support.

Reply:
Changed the name of method as `isWhocolorLang`